src/components/projects-summary/projects-summary.html

```diff
@@ -88,4 +88,25 @@ <h2 class="secondary">Resources</h2>
       </ul>
     </div>
   </div>
+  <div ng-if="$ctrl.recentlyViewedItems && $ctrl.recentlyViewedItems.length > 0">
+    <h2 class="primary">Recently Viewed</h2>
+    <div class="services-view">
+      <div class="services-view-container">
+        <div class="services-items-inner">
+          <div class="services-items-inner-shadow-covers">
+            <a href="" class="services-item" ng-repeat="item in $ctrl.recentlyViewedItems track by item.resource.metadata.uid"
```

Comment:
Better to use the UID filter

```
track by (item.resource | uid)
```

It avoids duplicates in a repeater errors if UID is not set for some reason.